### PR TITLE
fix(sqs_utils.js): update sns-sqs support

### PR DESCRIPTION
**Pull request by ranrib**
Signature fields are not always coming through

### CODE REVIEW

File: src/resource_utils/sqs_utils.js [hunk 1]
```diff
@@ -25,9 +25,7 @@ function getSNSTrigger(messages) {
                 'MessageId' in body &&
                 'TopicArn' in body &&
                 'Message' in body &&
-                'Timestamp' in body &&
-                'SignatureVersion' in body &&
```
Comment (haimrait) [inline src/resource_utils/sqs_utils.js:29]: Why did you remove them?
someone else uses them?

Comment (ranrib) [reply src/resource_utils/sqs_utils.js:29]: This is used to identify the message within SQS got the SNS signature. in SNS FIFO both signature fields do not appear and that's why they are not crucial (TopicArn, MessageId, and the rest are a good indicator for SNS usage as well)

**Review by haimrait — APPROVED** [review]
